client/my-sites/domains/components/domain-warnings/pending-gsuite-tos-notice-action.js

```diff
@@ -47,6 +47,12 @@ PendingGSuiteTosNoticeAction.propTypes = {
 	severity: PropTypes.string.isRequired,
 	siteSlug: PropTypes.string.isRequired,
 	user: PropTypes.string.isRequired,
+	cta: PropTypes.string,
+	isCompact: PropTypes.bool,
+};
+
+PendingGSuiteTosNoticeAction.defaultProps = {
+	compact: true,
```

Comment:
Shouldn't this be `isCompact` instead of `compact`?

Reply:
Yep, thanks for pointing it out.